db/seeds.rb

```diff
@@ -1,46 +1,227 @@
+withdrawn_proposal.speakers.create(speaker_name: speaker_3.name, speaker_email: speaker_3.email, bio: "I am the best speaker in the entire world!", user_id: speaker_3.id, event_id: seed_event.id)
+
+# Proposal Tags
+submitted_proposal_1.taggings.create(proposal_id: submitted_proposal_1.id, tag: "intermediate")
+submitted_proposal_2.taggings.create(proposal_id: submitted_proposal_2.id, tag: "intermediate")
+waitlisted_proposal.taggings.create(proposal_id: waitlisted_proposal.id, tag: "beginner")
+soft_accepted_proposal.taggings.create(proposal_id: soft_accepted_proposal.id, tag: "beginner")
+soft_rejected_proposal.taggings.create(proposal_id: soft_rejected_proposal.id, tag: "beginner")
+withdrawn_proposal.taggings.create(proposal_id: withdrawn_proposal.id, tag: "advanced")
+
+# Reviewer Tags
+submitted_proposal_1.taggings.create(proposal_id: submitted_proposal_1.id, tag: "beginner", internal: true)
+submitted_proposal_2.taggings.create(proposal_id: submitted_proposal_2.id, tag: "beginner", internal: true)
+waitlisted_proposal.taggings.create(proposal_id: waitlisted_proposal.id, tag: "beginner", internal: true)
+soft_accepted_proposal.taggings.create(proposal_id: soft_accepted_proposal.id, tag: "beginner", internal: true)
+soft_rejected_proposal.taggings.create(proposal_id: soft_rejected_proposal.id, tag: "intermediate", internal: true)
+withdrawn_proposal.taggings.create(proposal_id: withdrawn_proposal.id, tag: "advanced", internal: true)
+
+# Ratings
+submitted_proposal_1.ratings.create(proposal_id: submitted_proposal_1.id, user_id: organizer.id, score: 4)
+submitted_proposal_1.ratings.create(proposal_id: submitted_proposal_1.id, user_id: reviewer.id, score: 3)
+submitted_proposal_1.ratings.create(proposal_id: submitted_proposal_1.id, user_id: track_director.id, score: 5)
+
+submitted_proposal_2.ratings.create(proposal_id: submitted_proposal_2.id, user_id: organizer.id, score: 4)
+submitted_proposal_2.ratings.create(proposal_id: submitted_proposal_2.id, user_id: speaker_reviewer.id, score: 2)
+
+waitlisted_proposal.ratings.create(proposal_id: waitlisted_proposal.id, user_id: track_director.id, score: 3)
+waitlisted_proposal.ratings.create(proposal_id: waitlisted_proposal.id, user_id: organizer.id, score: 3)
+
+soft_accepted_proposal.ratings.create(proposal_id: soft_accepted_proposal.id, user_id: organizer.id, score: 4)
+soft_accepted_proposal.ratings.create(proposal_id: soft_accepted_proposal.id, user_id: reviewer.id, score: 3)
+soft_accepted_proposal.ratings.create(proposal_id: soft_accepted_proposal.id, user_id: track_director.id, score: 4)
+soft_accepted_proposal.ratings.create(proposal_id: soft_accepted_proposal.id, user_id: speaker_reviewer.id, score: 5)
 
-# Proposals
+soft_rejected_proposal.ratings.create(proposal_id: soft_rejected_proposal.id, user_id: organizer.id, score: 1)
```

Comment:
See if you can remove the proposal_id from the create call.  Since you're chaining the create off of the proposal initially it should get it for free.  Look for other similar patterns in this file.  Essentially you want to pass the minimal data possible.  

Also you pass objects vs ids such as:
soft_rejected_proposal.ratings.create(user: organizer, score: 1)